lib/mcb/commands/providers/show.rb

```diff
@@ -27,6 +27,6 @@
        'address2', 'address3', 'address4', 'postcode', 'telephone'
 
     puts "\nProvider Courses:"
-    tp provider.courses
+    provider.courses.map { |course| puts Terminal::Table.new rows: MCB::CourseShow.new(course).to_h }
```

Comment:
Can we add a switch to make this optional? I like the compact version of this, and usually manage the wrapping by piping to a pager. Maybe we should invoke a pager automatically like git does.

Reply:
I've made the behaviour optional.